delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StoragePluginModule.scala

```diff
@@ -125,7 +126,9 @@ object StoragePluginModule extends ModuleDef {
     )
   }
 
-  make[StorageScopeInitialization]
+  make[StorageScopeInitialization].from { (storages: Storages, serviceAccount: ServiceAccount) =>
```

Comment:
This is probably not needed if it is just used on `ScopeInitialization`. You can very well just have:
```scala
many[ScopeInitialization].add{...}
```